app/employee-app/backend/modules/database/db_functions.bal

```diff
@@ -50,31 +51,64 @@ public isolated function getEmployee(string email) returns Employee|error? {
-# + limit - Number of records to retrieve
+# + 'limit - Number of records to retrieve
 # + offset - Number of records to offset
-# + return - Returns the basic information of employees or an error
+# + return - Basic information of employees or an error
 public isolated function getEmployees(EmployeeFilter filters, int 'limit, int offset)
     returns Employee[]|error {
 
-    stream<DBEmployee, error?> resultStream = databaseClient->query(getEmployeesQuery(filters, 'limit, offset));
-    return from DBEmployee dbEmployeeInfo in resultStream
+    stream<EmployeeDb, error?> resultStream = databaseClient->query(getEmployeesQuery(filters, 'limit, offset));
+    return from EmployeeDb EmployeeDbInfo in resultStream
         select {
-            employeeId: dbEmployeeInfo.employeeId,
-            workEmail: dbEmployeeInfo.workEmail,
-            firstName: dbEmployeeInfo.firstName,
-            lastName: dbEmployeeInfo.lastName,
-            employeeThumbnail: dbEmployeeInfo.employeeThumbnail,
-            location: dbEmployeeInfo.location,
-            startDate: dbEmployeeInfo.startDate,
-            leadEmail: dbEmployeeInfo.leadEmail,
-            finalDayOfEmployment: dbEmployeeInfo.finalDayOfEmployment,
-            employeeStatus: dbEmployeeInfo.employeeStatus,
-            designation: dbEmployeeInfo.designation,
-            employmentType: dbEmployeeInfo.employmentType,
-            team: dbEmployeeInfo.team,
-            businessUnit: dbEmployeeInfo.businessUnit,
-            unit: dbEmployeeInfo.unit,
-            jobBand: dbEmployeeInfo.jobBand,
-            lead: dbEmployeeInfo.lead == 1
+            employeeId: EmployeeDbInfo.employeeId,
+            workEmail: EmployeeDbInfo.workEmail,
+            firstName: EmployeeDbInfo.firstName,
+            lastName: EmployeeDbInfo.lastName,
+            employeeThumbnail: EmployeeDbInfo.employeeThumbnail,
+            location: EmployeeDbInfo.location,
+            startDate: EmployeeDbInfo.startDate,
+            leadEmail: EmployeeDbInfo.leadEmail,
+            finalDayOfEmployment: EmployeeDbInfo.finalDayOfEmployment,
+            employeeStatus: EmployeeDbInfo.employeeStatus,
+            designation: EmployeeDbInfo.designation,
+            employmentType: EmployeeDbInfo.employmentType,
+            team: EmployeeDbInfo.team,
+            businessUnit: EmployeeDbInfo.businessUnit,
+            unit: EmployeeDbInfo.unit,
+            jobBand: EmployeeDbInfo.jobBand,
+            lead: EmployeeDbInfo.lead == 1
         };
 }
+
+# Retrieve organizational structure.
+#
+# + filter - Filter objects containing the filter criteria for the query
+# + 'limit - Number of records to retrieve
+# + offset - Number of records to offset
+# + return - List of business units
+public isolated function getOrgStructure(orgStructureFilter filter, int 'limit, int offset)
+    returns OrgStructure|error {
+
+    OrgStructure orgStructure = {
+        businessUnits: []
+    };
```

Comment:
```suggestion
      BusinessUnit [] businessUnits = [];
```

Reply:
OrgStructure type is a redundant type. We can directly return BusinessUnit[] and map it to the return type from the service level.

Reply:
Did this because the method name provides an idea of an organization structure but the response gives and idea of business units. So wrapped the nisa units with a new record called `OrgStructure`.